### DISCUSSION TITLE
Do you think ControlNet can accomplish the following tasks?

### DISCUSSION THREAD

**Discussion by JaosonMa** [lllyasviel/ControlNet#514] [Ideas]
![image](https://github.com/lllyasviel/ControlNet/assets/23551774/51b9f4eb-ac0e-4000-a695-9ec0ea295aff)

I want to achieve the conversion of hand-drawn physical circuit diagrams into vector images similar to those created by engineering software. Do you think ControlNet can accomplish this task?
@lllyasviel

**Comment by geroldmeisinger** [comment]
I think this can be better achieved with classic algorithmic approaches or specialized models. Stable Diffusion is for generating images.

crosspost of https://github.com/lllyasviel/ControlNet/issues/526
